modin/pandas/series.py

```diff
@@ -1416,20 +1404,17 @@ def pow(self, other, level=None, fill_value=None, axis=0):  # noqa: PR01, RT01,
             new_other, level=level, fill_value=None, axis=axis
         )
 
-    def prod(
+    @_inherit_docstrings(pandas.Series.prod, apilink="pandas.Series.prod")
+    def _prod(
         self,
-        axis=None,
-        skipna=True,
-        level=None,
-        numeric_only=None,
-        min_count=0,
+        axis,
+        skipna,
+        level,
+        numeric_only,
+        min_count,
         **kwargs,
-    ):  # noqa: PR01, RT01, D200
```

Comment:
For some functions, you do not add a docstring, but leave pragmas for `flake8`. (`_reindex`, for example, 1473 line)
What is the difference between these functions?

Reply:
I think some of the methods were changed by Devin and some by me, that's why we did it differently.
I'll add the decorator for `_reindex`. Non-underscore methods having those pragmas are probably some leftover which wasn't removed when `_inherit_docstrings()` was made to support decorating classes. I would rather fix that in a separate issue/PR.

Reply:
Could you create an issue for this?

Reply:
https://github.com/modin-project/modin/issues/4679

Reply:
Thanks!